mailpoet/mailpoet
# Update block editor in the form editor [MAILPOET-4029][MAILPOET-3914]

[MAILPOET-4029] [MAILPOET-3914]

## Testing instructions
Info: This PR updates internals of the form editor.

1. Go to MailPoet > Forms
2. Please perform exploratory testing of the form editor, test complete functionalities.
3. There are 2 visible changes in UI/functionality that you should check also:

a. **New UI for color selection**
<img width="266" alt="Screenshot 2022-01-06 at 16 42 15" src="https://user-images.githubusercontent.com/1082140/148415004-0b3b3a00-892f-4faf-aab5-268a41193dcb.png">

b. **It is possible to set padding also to columns wrapper.**
<img width="286" alt="Screenshot 2022-01-06 at 16 42 49" src="https://user-images.githubusercontent.com/1082140/148415268-7135a5e6-1483-47cb-a667-d5a91f81b63b.png">

[MAILPOET-4029]: https://mailpoet.atlassian.net/browse/MAILPOET-4029?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MAILPOET-3914]: https://mailpoet.atlassian.net/browse/MAILPOET-3914?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
